embeddings/embedding/sklearn_embedding.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any, Dict, Optional, Union
+
+import pandas as pd
+from numpy import typing as nptyping
+from sklearn.base import BaseEstimator as AnySklearnVectorizer
+
+from embeddings.embedding.embedding import Embedding
+
+
+class SklearnEmbedding(Embedding[Union[pd.Series, nptyping.NDArray[Any]], pd.DataFrame]):
```

Comment:
However, I'd create a custom type for Union[pd.Series, nptyping.NDArray[Any], e.g. arraylike or something like that. Sklearn documentation for CountVectorizer defines the input as follows: 

> An iterable which generates either str, unicode or file objects.